Configuration.props

```diff
@@ -33,6 +33,7 @@
     <AutoProvision Condition=" '$(AutoProvision)' == '' ">False</AutoProvision>
     <AutoProvisionUsesSudo Condition=" '$(AutoProvisionUsesSudo)' == '' ">False</AutoProvisionUsesSudo>
     <XAInstallPrefix Condition=" '$(XAInstallPrefix)' == '' ">$(MSBuildThisFileDirectory)\bin\$(Configuration)\lib\xamarin.android\</XAInstallPrefix>
+    <LinuxMingwDependenciesRootDirectory Condition=" '$(LinuxMingwDependenciesRootDirectory)' == '' ">$(MSBuildThisFileDirectory)\bin\$(Configuration)\linux-mingw-deps</LinuxMingwDependenciesRootDirectory>
```

Comment:
I believe these outputs should go into `bin/Build$(Configuration)`, not `bin/$(Configuration)`.